test/api_test.gd

```diff
@@ -90,3 +97,75 @@ func test_entity_is_category() -> void:
 	assert_bool(sword.is_category(items.get_entity_id())).is_true()
 	assert_bool(sword.is_category(weapons.get_entity_id())).is_true()
 	assert_bool(sword.is_category(quests.get_entity_id())).is_false()
+
+func test_import_duplicates() -> void:
+	Pandora.set_context_id(TEST_DIR)
+	Pandora._clear()
+	Pandora.load_data()
+	Pandora.create_category("new_category2")
+	Pandora.create_entity("new_entity", Pandora.create_category("new_category"))
+	Pandora.save_data()
+	var data_file = ProjectSettings.globalize_path("res://" + TEST_DIR + "/data.pandora")
+	var imported_count: int = Pandora.import_data(data_file)
+
+	DirAccess.remove_absolute("res://" + TEST_DIR + "/data.pandora")
```

Comment:
@mrfelipemartins  This stuff here is no longer required, because we auto clean up after each test run: https://github.com/bitbrain/pandora/blob/godot-4.x/test/api_test.gd#L20

Reply:
Maybe this should be after_test() no? Bc test_import_empty_data_file will fail if I don't clean up before this test case starts. after() is only cleaning after the entire test suite is over

Reply:
Ah, makes sense. Feel free to refactor that!

Reply:
Done.